src/components/language/language.service.ts

```diff
@@ -243,6 +244,12 @@ export class LanguageService {
           isPublic: false,
           isOrgPublic: false,
         },
+        {
+          key: 'canDelete',
```

Comment:
This property is not technically needed since all we're checking is the canDelete permission, but agreed with Michael to keep this in for consistency